packs/spell-effects/effect-hand-of-the-apprentice.json

```diff
@@ -0,0 +1,69 @@
+{
+    "_id": "3pb8XJTwAjkx8l4o",
+    "img": "systems/pf2e/icons/spells/hand-of-the-apprentice.webp",
+    "name": "Effect: Hand of the Apprentice",
```

Comment:
Should be named "Spell Effect: Hand of the Apprentice".

Reply:
Should be corrected now. Please check when you get a moment.

Thanks.